src/nav/browser.yml

```diff
@@ -1,66 +1,98 @@
+      - title: SPA Compatibility and requirements
+        path: /docs/browser/single-page-app-monitoring/get-started/spa-compatibility-requirements
+      - title: Install SPA monitoring
+        path: /docs/browser/single-page-app-monitoring/get-started/install-single-page-app-monitoring
+      - title: View SPA data in UI
+        path: /docs/browser/single-page-app-monitoring/use-spa-data/view-spa-data-browser-ui
+      - title: SPA monitoring features
+        path: /docs/browser/new-relic-browser/browser-pro-features/spa-monitoring
+      - title: SPA data collection
+        path: /docs/browser/single-page-app-monitoring/use-spa-data/spa-data-collection
+      - title: Query SPA data
+        path: /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/browserspa-nrql-query-examples
+      - title: SPA APIs
+        path: /docs/browser/new-relic-browser/browser-apis/using-browser-apis#spa-api
+  - title: Best practices guide
+    path: /docs/browser/browser-monitoring/guides/browser-monitoring-best-practices-guide
+  - title: Page load timing
+    pages:
+      - title: Page load timing process
+        path: /docs/browser/new-relic-browser/page-load-timing-resources/page-load-timing-process
+      - title: PageViewTiming event
+        path: /docs/browser/new-relic-browser/page-load-timing-resources/pageviewtiming-async-or-dynamic-page-details
+      - title: Instrumentation for browser monitoring
+        path: /docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring/
+      - title: Compare synthetic and browser page performance
+        path: /docs/browser/new-relic-browser/page-load-timing-resources/compare-page-load-performance-browser-synthetics
+      - title: New Relic cookies used by browser
+        path: /docs/browser/browser-monitoring/page-load-timing-resources/new-relic-cookies-used-browser
+      - title: Cookie collection and session tracking
+        path: /docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking
+      - title: Cached pages
+        path: /docs/browser/new-relic-browser/page-load-timing-resources/cached-pages
   - title: Installation
     pages:
-      - title: Install browser monitoring
+      - title: Install the browser agent
         path: /docs/browser/browser-monitoring/installation/install-browser-monitoring-agent
       - title: Update the browser agent
         path: /docs/browser/new-relic-browser/installation/update-browser-agent
-      - title: Disable browser monitoring
-        path: /docs/browser/new-relic-browser/installation/disable-browser-monitoring
-      - title: Delete apps from New Relic
-        path: /docs/browser/new-relic-browser/installation/delete-apps-new-relic-browser
+      - title: Compatibility and requirements
+        path: /docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring
   - title: Configuration
```

Comment:
@paperclypse Thoughts on this title? I tried other variations, and they were too long or didn't match well with the other tree titles :(

Reply:
@a-sassman I think it's fine. It makes it pretty clear what the doc is all about.